package.json

```diff
@@ -26,7 +26,6 @@
   },
   "dependencies": {
     "@docusaurus/core": "^2.4.0",
-    "@docusaurus/plugin-google-tag-manager": "^2.4.0",
```

Comment:
This is already in the theme and doesn't need to be added as another dep.
https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-google-gtag#installation